modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -9,35 +9,39 @@ Runtime Fabric can be shared across multiple environments. After installation, c
 
 Applications deployed to a production environment must be deployed to an instance of Runtime Fabric that is separate from nonproduction applications.
 
-By default, inbound traffic is disabled to prevent consuming unnecessary resources. To allow Mule applications or API gateways to listen on inbound connections, enable inbound traffic.
+By default, inbound traffic is disabled to prevent consuming unnecessary resources. To allow Mule applications or API gateways to listen on inbound connections, you must enable inbound traffic.
 
-== Step 1: Assign Required Roles and Permissions
-Ensure you have the necessary permissions for the Runtime Fabric environment you want to use for applications:
+Make sure you have completed the prerequisite network-related tasks in xref:install-prereqs.adoc#infrastructure-requirements[Infrastructure Requirements] before enabling inbound traffic on Runtime Fabric.
 
-. From Anypoint Platform, select *Access Management*.
-+
-If you do not have access to *Access Management*, ask your organization admin (system administrator) to provide the "Manage Runtime Fabrics" permission to your user. If you have access, execute the following steps to add the permission:
-. Select *Users*.
-. Locate the row containing your username and then select the blue link in the *Username* column.
-. In the *Runtime Manager* tab, enable the *Manage Runtime Fabrics* permission for your Runtime Fabric environment.
-+
-[NOTE]
-It can take up to five minutes for any permissions you add to propagate.
-
-== Step 2: Configure Inbound Traffic Using Runtime Manager
+== Step 1: Configure Inbound Traffic Using Anypoint Runtime Manager
 
 . Navigate to Runtime Manager and select *Runtime Fabric*.
 . Select the name of your Runtime Fabric to open the management page.
 . In the *Associated Environments* tab, ensure that your Runtime Fabric is associated with the environments in which applications are to be deployed.
 . Select *Inbound Traffic* and toggle *Enable inbound traffic* to on.
 +
 [NOTE]
-If you disable the Inbound Traffic tab, changes can take several minutes to complete.
+If you disable the *Inbound Traffic* tab, changes can take several minutes to complete.
+
+. In the *Basic Configuration* section, select one of the following Protocol options:
+
+* *Accept HTTPS requests…*
++
+Then select one of the following options:
+
+** *and redirect HTTP to HTTPS*
+** *and also accept HTTP requests*
+** *and drop HTTP requests*
+
+* *Accept HTTP requests only*
++
+Use the *Add Domain* button to specify domains. All specified domains are displayed in the *Domains* section.
++
+If you select this option, all certs are deleted.
```

Comment:
I think it may be distracting to include this info here, because it's only relevant in a specific and rare situation, which is why we warn about it in the UI.

Reply:
OK - thanks! I removed this.